pkg/apis/core/types.go

```diff
@@ -419,13 +419,18 @@ type PersistentVolumeClaimSpec struct {
 	// This is a beta feature.
 	// +optional
 	VolumeMode *PersistentVolumeMode
-	// This field can be used to specify either:
-	// * An existing VolumeSnapshot object (snapshot.storage.k8s.io/VolumeSnapshot)
-	// * An existing PVC (PersistentVolumeClaim)
-	// In order to use either of these DataSource types, the appropriate feature gate
-	// must be enabled (VolumeSnapshotDataSource, VolumePVCDataSource)
+	// This field can be used to specify that the volume should initially contain
```

Comment:
maybe something like:

```
	// This field can be used to specify either:
	// * An existing VolumeSnapshot object (snapshot.storage.k8s.io/VolumeSnapshot - Beta)
	// * An existing PVC (PersistentVolumeClaim)
        // * Any external CRD/Object that implements data population (anyVolumeDataSource - Alpha)
	// In order to use VolumeSnapshot object types, the appropriate feature gate
	// must be enabled (VolumeSnapshotDataSource or AnyVolumeDataSource)
	// If the provisioner or an external controller can support the specified data source, it will create
	// a new volume based on the contents of the specified data source.
	// If the specified data source is not supported, the volume will
	// not be created and the failure will be reported as an event.
	
```